### PR TITLE
Documentation

### LINKED ISSUE #63: add documentation to classes which are to be implemented

### PR DESCRIPTION
**Pull request by Bartzi**
adds some more documentation and fixes #63

### CODE REVIEW

File: length/function.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def internal_backward(self, inputs, gradients):
         """
         :param inputs: is a tuple of numpy arrays
         :param gradients: is a tuple of numpy arrays
-        :returns gradients as tuple of numpy arrays (first element is gradient with respect to input)
+        :returns gradients as tuple of numpy arrays (first elements are gradients with respect to input)
```
Comment (Jopyth) [inline length/function.py]: first element is an array of gradients with respect to input?

**Review by Jopyth — APPROVED** [review]
:eyeglasses:

File: length/function.py (hunk 1 above)
Comment (Bartzi) [reply length/function.py]: no, hmm I wanted to say the first n elements are gradients with respect to the inputs (with n being the number of inputs)... any better idea?

Comment (Jopyth) [reply length/function.py]: Oh right, never mind.

**Pull request merged**